docs/index.md

```diff
@@ -29,4 +29,31 @@ See the relationship between the connector files (in blue) and the Tableau **Con
 You need to do these things before you start:
 
 - Install the ODBC or JDBC driver that you’ll use with the connector you’ll create.
-- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
\ No newline at end of file
+- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
+
+## Using a Connector
+You must start Tableau Desktop or Server with a special command line argument which tells Tableau where to find your Connector. See [Share Your Connector]({{ site.baseurl }}/docs/share) for more information.
```

Comment:
Replace first sentence: You must start Tableau Desktop or Tableau Server with a special command line argument that tells Tableau where to find your connector.

Reply:
The content is great!  For the most part, the changes I suggest are just to make the content consistent with our style guide.